drivers/main-device.js

```diff
@@ -336,7 +332,7 @@ module.exports = class mainDevice extends Homey.Device {
     async getComponentValue(val, components) {
         const comp = components.find((el, id) => el.componentType === val);
         if (comp) {
-            return comp.value === 'HIGH';
+            return comp.value === 'HIGH' || comp.value === 'ON';
```

Comment:
Don't know what needs to be done to support "WAITING" for HEATER component value.